server/ansible/roles/pbench-repo-install/vars/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+baseurl: "{{ 'http://coprbe.devel.redhat.com/results/ndokos/pbench/rhel-8.dev-$basearch/'
+                 if ansible_distribution == 'RedHat' and ansible_distribution_major_version == '8'
+                 else
+             'https://copr-be.cloud.fedoraproject.org/results/ndokos/pbench/epel-7-$basearch/'
+                 if ansible_distribution == 'RedHat' and ansible_distribution_major_version == '7'
+                 else
+             'https://copr-be.cloud.fedoraproject.org/results/ndokos/pbench/epel-6-$basearch/'
+                 if ansible_distribution == 'RedHat' and ansible_distribution_major_version == '6'
+                 else
+             'https://copr-be.cloud.fedoraproject.org/results/ndokos/pbench/fedora-$releasever-$basearch/'
+                 if ansible_distribution == 'Fedora'
+                 else
+             '' }}"
+
+testbaseurl: "{{ 'http://coprbe.devel.redhat.com/results/ndokos/pbench-test/rhel-8.dev-$basearch/'
```

Comment:
Can we also remove the test versions of things?

Reply:
Yes.